tests/main/classic-prepare-image/task.yaml

```diff
@@ -48,10 +50,8 @@ prepare: |
     systemd-run --unit fakedevicesvc fakedevicesvc localhost:11029
 
 restore: |
-    if [ "$TRUST_TEST_KEYS" = "false" ]; then
-        echo "This test needs test keys to be trusted"
-        exit
-    fi
+    test -f .skip && exit 0
```

Comment:
sounds like this could be some sort of helper `tests.exec is-skipped && exit 0` ? and a similar helper `tests.exec skip "this is the reason for skipping"`

Reply:
updated